proto/penumbra/penumbra/core/component/compact_block/v1/compact_block.proto

```diff
@@ -28,6 +28,8 @@ message CompactBlock {
   bool app_parameters_updated = 9;
   // Updated gas prices, if they have changed.
   fee.v1.GasPrices gas_prices = 10;
+  // The epoch index
+  uint64 epoch_index = 11;
```

Comment:
It would be grand to have a more descriptive docstring here, incorporating some of the context from #2689, that speaks to what this value represents, as well as why and how it might be used. Unfortunately, I'm a bit at a loss to suggest specific phrasing.

Reply:
I was also at a loss on this one ... maybe @hdevalence or perhaps @plaidfinch could shed some light on the "epoch index" for future readers of our documentation strings?

Reply:
This is the counter that identifies/index each epoch similar to the height of a block

Reply:
but for epochs